### PR TITLE
1242 - Move operations to separate modals

### LINKED ISSUE #1242: Move operations to separate modals
Currently, the Settings modal contains buttons for operations (e.g. import/export, Kaltura upload, and optimizing thumbnails) alongside settings fields. These operations are not affected by saving/submitting the settings modal, so it makes more sense to move them into separate modals.

* Add a kebab menu icon (three dots) to the Tapestry menubar.
* When the user clicks it, it should show a dropdown with options for different groups of operations (currently: export/duplicate, Kaltura, other).
* Each group of operations should open its own modal when clicked.

### PR DESCRIPTION
**Pull request by zifgu**
## Description
Currently, the Settings modal contains buttons for operations (e.g. import/export, Kaltura upload, and optimizing thumbnails) alongside settings fields. These operations are not affected by saving/submitting the settings modal, so we would like to move them into separate modals.

## Changes
* Adds a menu button to the Tapestry menubar that opens a dropdown with different Tapestry-wide operations (currently: Export/Duplicate Tapestry, Other Operations).
    * When a dropdown item is clicked, it opens the corresponding modal
* Moves Export/Duplicate functionality from Settings into its own modal
* Moves Thumbnail optimization functionality from Settings into its own modal (Other Operations)
* Fixes failing Cypress tests as a result of the changes

## Screenshot
New Tapestry menubar:
image

## Issue Linkage
Closes #1242 
## PR Dependency
Depends on: N/A
## Automated Testing
N/A

### CODE REVIEW

**Review by legendword — APPROVED** [review]
The code changes look great! The functionalities are still working. Just some minor details and a style change that makes buttons in the dropdown list have a nicer look while active (pressed down). I made a commit that includes these changes. @zifgu you can take a look and let me know if there's any issues with them. If not it is ready for final review.

**Comment by zifgu** [comment]
> Just some minor details and a style change that makes buttons in the dropdown list have a nicer look while active (pressed down).

The new style looks really good! Thank you for changing the test name too. As soon as I confirm the CI tests pass, I'll mark this as ready for final review.

File: templates/vue/src/components/Toolbar/TapestryOperationsButton.vue [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      Other Operations
+    </b-dropdown-item-button>
+    <export-duplicate-modal
+      :show="openOperation === names.EXPORTDUPLICATE"
+      @close="close(names.EXPORTDUPLICATE)"
+    ></export-duplicate-modal>
+    <other-operations-modal
+      :show="openOperation === names.OTHEROPERATIONS"
+      @close="close(names.OTHEROPERATIONS)"
+    ></other-operations-modal>
```
Comment (wynnset) [inline templates/vue/src/components/Toolbar/TapestryOperationsButton.vue:28]: Best if these are nested outside of `b-dropdown`

**Review by wynnset — CHANGES_REQUESTED** [review]
It's best not to use "tapestry" in the name of things unless it's really necessary. In this case, "Tapestry Operations" should be just "Operations". Also just one comment below:

File: templates/vue/src/components/Toolbar/TapestryOperationsButton.vue (hunk 1 above)
Comment (zifgu) [reply templates/vue/src/components/Toolbar/TapestryOperationsButton.vue]: Sounds good, that makes sense. 7dba59b should fix this.

**Review by wynnset — APPROVED** [review]

**Pull request merged**